tf_big/cc/kernels/big_tensor.h

```diff
@@ -45,11 +51,10 @@ typedef Matrix<mpz_class, Dynamic, Dynamic> MatrixXm;
 namespace tf_big {
 
 struct BigTensor {
- public:
-  BigTensor(){};
+  BigTensor() {}
   BigTensor(const BigTensor& other);
-  BigTensor(mpz_class m);
-  BigTensor(const MatrixXm& mat);
+  explicit BigTensor(mpz_class m);
```

Comment:
what does `explicit` do?

Reply:
https://google.github.io/styleguide/cppguide.html#Implicit_Conversions, basically just makes sure the types passed into the constructor aren't implicitly converted.